v3.1/sites-available/000-default.conf

```diff
@@ -6,9 +6,15 @@
 
   LogLevel ${APACHE_LOGLEVEL}
   ErrorLog ${APACHE_ERRORLOG}
-  CustomLog ${APACHE_ACCESSLOG}
+  CustomLog ${APACHE_ACCESSLOG} env=!nologging
   CustomLog ${APACHE_PERFLOG}
 
+  <Location "/healthz">
+    SetEnv nologging
+    SetHandler server-status
```

Comment:
I wouldn't expose the `server-status` as it exposes a lot of data and rather use the `mod_rewrite` to return (or redirect with `[R=200]`) a 200 response.